public/static/app/perfdiscovery/PerfDiscoveryService.js

```diff
@@ -50,9 +50,9 @@ mciModule.factory('PerfDiscoveryService', function($q, ApiV1, ApiTaskdata) {
   }
 
   // Calculates ratio for given test result `item` and `baseSpeed`
-  function ratio(item, baseSpeed) {
-    var ratio = item.speed / baseSpeed
-    if (item.speed >= 0) {
+  function ratio(speed, baseSpeed) {
```

Comment:
The comment needs to be updated for the parameter names.

I think that it should also refer to the comment on line 58:
```
 Negatives mean latency in ms. Invert the ratio so higher is better.
```